app/controllers/posts_controller.rb

```diff
@@ -21,6 +21,9 @@ def create
   def update
     authorize @post, :owner?
     if @post.update(post_params)
+      if params[:post][:remove_banner] == "1"
```

Comment:
Tev neobligati bija jāparraksta kodu, es tev pagajušajā reizē vnk pastastiju kā vareja vieglak uztaisit. Bet ok! Ļoti labi!

Reply:
a nu katra gadijuma to metodi ko pastastiji man daudz labak patika, tapec pat nebija zel parrakstit 😄